models/app.models.js

```diff
@@ -47,10 +47,26 @@ function insertComment(username, body, review_id) {
     .then(({ rows }) => rows[0]);
 }
 
+function updateReview(inc_votes, review_id) {
+  return db
+    .query(
+      "UPDATE reviews SET votes = votes + $1 WHERE review_id = $2 RETURNING *;",
+      [inc_votes, review_id]
+    )
+    .then(({ rows }) => {
+      const review = rows[0];
+      if (!review) {
+        return checkExists("reviews", "review_id", review_id);
```

Comment:
Think you dont need this. 
If its an empty array, its because it couldnt find the article id in the db. You dont need to query the db again to check the same thing.